im.due.Panamaram.yaml

```diff
@@ -0,0 +1,36 @@
+app-id: im.due.Panamaram
+runtime: org.kde.Platform
+runtime-version: '6.9'
+sdk: org.kde.Sdk
+base: io.qt.PySide.BaseApp
+base-version: '6.9'
+
+command: run-app.sh
+
+finish-args:
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=wayland
+  - --device=dri
+
+modules:
+  - name: panamaram
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix=/app --no-warn-script-location flatpak/pypi/*.whl
```

Comment:
build it here don't hide a binary wheel inside an archive

Reply:
why did you put every binary wheel in here?

<img width="862" height="618" alt="image" src="https://github.com/user-attachments/assets/860b0d98-40f5-4584-8120-1e93b499c144" />

Reply:
these all needs to be built here from their original sources not whatever random things you packed into your archive

use https://github.com/flatpak/flatpak-builder-tools/tree/master/pip